ios/PurchasesHybridCommon/PurchasesHybridCommon/CommonFunctionality.swift

```diff
@@ -41,6 +41,10 @@ import RevenueCat
         set {
             if let value = newValue {
                 var url: URL?
```

Comment:
While you're here, can you make this a `let`?

Reply:
will do